docs/notebooks/gis.ipynb

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
<p>Il faudrait idéalement ajouter 2-3 informations sur comment c'est fait.</p><p> </p><p><code>Currently, watershed delineation uses HydroBASINS (hybas_na_lev01-12_v1c) and can work in any location in North America. The process involves assessing all upstream sub-basins from a specified pour point and consolidating them into a unified watershed.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>